source/settings/plugin/fts-plugin.rst

```diff
@@ -0,0 +1,453 @@
+
+Language specific filter chains can be specified with ``fts_filters_<lang>`` (e.g. ``fts_filters_en``).
+
+List of available filters:
+
+``lowercase``:
+
+  Change all text to lower case. Supports UTF8, when compiled with libicu and
+  the library is installed. Otherwise only ASCII characters are lower cased.
+
+``stopwords``:
+
+  Filter certain common and short words, which are usually useless for
+  searching.
+
+  Settings: ``stopwords_dir``, path to the directory containing stopword files.
+
+  Stopword files are looked up in ``”<path>”/stopwords_<lang>.txt``.
+
+  Currently Dovecot distributes stopword files for the following languages:
+
+  +---------------+----------+
+  | Language Code | Language |
+  +===============+==========+
+  | en            | English  |
+  +---------------+----------+
+  | fi            | Finnish  |
+  +---------------+----------+
+  | fr            | French   |
+  +---------------+----------+
+
+  More can be obtained from the Apache Lucene project or the snowball stemmer
+  source.
+
+  Stopword language files are also available from
+  https://github.com/stopwords-iso/.
+
+``snowball``:
+
+  Stemming tries to convert words to a common base form. A simple example is
+  converting “cars” to “car”.
+
+  This stemmer is based on the Snowball stemmer library.
+  
+  Supported languages:
+
+  +---------------+---------------------------------------+
+  | Language Code | Language                              |
+  +===============+=======================================+
+  | da            | Danish                                |
+  +---------------+---------------------------------------+
+  | de            | German                                |
+  +---------------+---------------------------------------+
+  | en            | English                               |
+  +---------------+---------------------------------------+
+  | es            | Spanish                               |
+  +---------------+---------------------------------------+
+  | fi            | Finnish                               |
+  +---------------+---------------------------------------+
+  | it            | Italian                               |
+  +---------------+---------------------------------------+
+  | nl            | Dutch                                 |
+  +---------------+---------------------------------------+
+  | no            | Norwegian (Bokmål & Nynorsk detected) |
+  +---------------+---------------------------------------+
+  | pt            | Portuguese                            |
+  +---------------+---------------------------------------+
+  | ro            | Romanian                              |
+  +---------------+---------------------------------------+
+  | ru            | Russian                               |
+  +---------------+---------------------------------------+
+  | sv            | Swedish                               |
+  +---------------+---------------------------------------+
```

Comment:
Should add Turkish and French here or somehow combine this with supported languages table.